app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java

```diff
@@ -39,11 +44,29 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         etBio = findViewById(R.id.etBio);
         tvEventCategoryDropdown = findViewById(R.id.tvEventCategoryDropdown);
         Button btnSignUp = findViewById(R.id.btnSignUp);
+        animatedConfetti = findViewById(R.id.animatedConfetti);
 
         setUpEventCategoryDropdown();
         btnSignUp.setOnClickListener(v -> signUpUser());
     }
 
+    private void startCheckAnimationLogo() {
```

Comment:
❓ What does this code do here?

Reply:
That function makes sure that the animation is done playing before moving on to the MainActivity - otherwise, the user wouldn't be able to see the confetti happen.